features/step_definitions/assertion_steps.rb

```diff
@@ -45,4 +49,18 @@
     fill_in 'Working hours', with: '9 hours'
     fill_in 'Date start', with: 'Tomorrow'
     fill_in 'Date finish', with: 'Next Year'
+end
+
+Then("I fill in company form with {string} information") do |email|
+    case email
+    when 'valid'
+        email = 'email@email.com' 
+    when 'invalid'
+        email = 'company1@me.com'
+    end
+    fill_in 'Name', with: 'Company 3'
+    fill_in 'Personal name', with: 'Angelica'
```

Comment:
Wouldn't `contact_name` be a more suitable attribute name?

Reply:
changed